lib/gui/app/modules/image-writer.js

```diff
@@ -306,19 +314,30 @@ exports.flash = (image, drives) => {
     drives,
     driveCount: drives.length,
     uuid: flashState.getFlashUuid(),
+    status: 'started',
+    flashInstanceUuid: flashState.getFlashUuid(),
     unmountOnSuccess: settings.get('unmountOnSuccess'),
-    validateWriteOnSuccess: settings.get('validateWriteOnSuccess')
+    validateWriteOnSuccess: settings.get('validateWriteOnSuccess'),
+    applicationSessionUuid: store.getState().toJS().applicationSessionUuid,
+    flashingWorkflowUuid: store.getState().toJS().flashingWorkflowUuid
   }
 
   analytics.logEvent('Flash', analyticsData)
 
   return exports.performWrite(image, drives, (state) => {
     flashState.setProgressState(state)
   }).then(flashState.unsetFlashingFlag).then(() => {
+    const { results } = flashState.getFlashResults()
+    const event = _.assign({
+      errors: results.errors,
+      devices: results.devices
+    },
+    analyticsData,
+    { status: 'finished' })
```

Comment:
Why explicitly add `{ status: 'finished' }`? Is this something that's left over from development?

Reply:
It's to make filtering easier on Mixpanel

Reply:
Can we put that on the first argument, which is the object literal, to avoid creating more objects than needed?